src/arch/riscv/isa/templates/vector_mem.isa

```diff
@@ -171,7 +171,7 @@ Fault
     if (fault != NoFault)
         return fault;
 
-    const size_t micro_vlmax = vtype_VLMAX(machInst.vtype8, vlen, true);
+    const size_t micro_vlmax = vlen / width_EEW(machInst.width);
     const size_t micro_elems = vlen / width_EEW(machInst.width);
```

Comment:
I think we can remove the `micro_elems` and only leave `micro_vlmax`

Reply:
Sure, I've pushed a new commit.